### ISSUE TITLE
Shift+Tab instead of Alt+F10

### ISSUE THREAD

**Issue by alecpl** [tinymce/tinymce#5289]
I find Alt+F10 as a keyboard shortcut not very user-friendly (i.e. you have to learn that it exists).

Please, consider adding possibility to do the same with Shift+Tab. So, when you're in the editor body, Shift+Tab will focus first button in toolbar, then another Shift+Tab will move the focus outside of the editor.

BTW, why Alt+F10 places focus on first toolbar button and not first menubar button?

This probably should be evaluated together with #5161 (and maybe with #4795).

**Comment by lostkeys** [comment]
Alt+F9 takes you to the menubar. I too find the Alt+F* unintuitive, as you have to know it beforehand.
However Shift+Tab defaults to tabbing backwards on MacOS and I don't think we want to override that platform convention.

**Comment by alecpl** [comment]
Yeah, I know about what normally Shift+Tab does. That's the point. It might be slightly unexpected, but if you wanted to leave the editor with Shift+Tab you have to just press it twice. Imo, still better than a requirement to know Alt+F* combinations.

**Comment by lostkeys** [comment]
I think I get what you mean – something like this?

![Shift-Tab-to-toolbar](https://user-images.githubusercontent.com/3122929/69946873-b8979280-14ec-11ea-8e73-91e3b961165f.gif)

**Comment by alecpl** [comment]
Yes.

**Comment by lostkeys** [comment]
I had a quick discussion internally with people knowing more than me and there is an a11y requirement that the forward tab and reverse tab order should be symmetric. So Shift + Tab should take you back to the previously focused element. Which kind of make sense if you can't see what is going on. It would feel very unpredictable if a reverse tab didn't go back to where you previously were.

The TinyMCE tab focus pattern is primarily built for screen reader use.

That said, there is always room for improvements and ideas and we would love more opinions and thoughts on this.

**Comment by perttumyry** [comment]
In Ubuntu (at least in version 22.04) Alt+F10 is the default OS shortcut for toggling window maximization state. So when using the default shortcuts Alt+F10 is handled by the OS and there is no way I can move focus to the TinyMCE toolbar using keyboard.
